src/routes/Send.tsx

```diff
@@ -577,26 +579,55 @@ export default function Send() {
-                                    {sentDetails()?.failure_reason}
-                                </p>
+                                <MegaEx />
+                                <LargeHeader centered>
+                                    {sentDetails()?.amount
+                                        ? "Payment Initiated"
+                                        : sentDetails()?.failure_reason}
+                                </LargeHeader>
+                                {/*TODO: add failure hint logic for different failure conditions*/}
+                                {/*<span>
+                                    <p class="text-xl text-center font-extrabold py-2 px-4 rounded-xl leading-[22px]">
+                                        {sentDetails()?.failure_reason}
+                                    </p>
+                                    <p class="text-center leading-[19px]">
+                                        failure hint
+                                    </p>
+                                </span>*/}
                             </Match>
                             <Match when={true}>
-                                <img
-                                    src={megacheck}
-                                    alt="success"
-                                    class="w-1/2 mx-auto max-w-[50vh]"
-                                />
+                                <MegaCheck />
+                                <LargeHeader centered>
+                                    {sentDetails()?.amount
+                                        ? "Payment Initiated"
+                                        : ""}
+                                </LargeHeader>
                                 <Amount
                                     amountSats={sentDetails()?.amount}
                                     showFiat
-                                    centered
+                                    align="center"
+                                    size="large"
+                                    icon="minus"
                                 />
+                                <hr class="w-16 bg-m-grey-400" />
+                                <div class="flex flex-row items-start gap-3">
+                                    <p class="text-m-grey-400 text-sm leading-[17px] text-center">
+                                        Transaction Fees
+                                    </p>
+                                    <div class="flex items-start gap-1">
+                                        <Amount
+                                            amountSats={
+                                                sentDetails()?.fee_estimate
+                                            }
+                                            align="right"
+                                            size="small"
+                                            showFiat
```

Comment:
size small align right:

small USD is missing a space
it should be 0 SATS, not just 0, right?

![Screenshot 2023-07-19 at 3 34 22 PM](https://github.com/MutinyWallet/mutiny-web/assets/543668/dd092d25-f6c6-401f-a001-e91ce3da648a)